src/applications/personalization/profile-2/tests/components/personal-information/PersonalInformation.delete-email-address.unit.spec.jsx

```diff
@@ -142,6 +155,19 @@ describe('Deleting email address', () => {
     expect(alert).to.contain.text(
       'We’re sorry. We couldn’t update your email address. Please try again.',
     );
+
+    // make sure that edit mode is not exited
+    let elementRemovedError = null;
```

Comment:
I need to write a blog post on this. Googling how to test that an element is _not_ removed from the DOM didn't help. So this is my solution. Essentially test that the `waitForElementToBeRemoved()` helper throws an error, which it will do if the element is not removed within its specified timeout (1000ms default, set to 75ms in this case). I confirmed that there tests failed before implementing my fix.